src/mirco_matrixsetup.cpp

```diff
@@ -5,26 +5,49 @@
 
 void MIRCO::MatrixGeneration::SetUpMatrix(Teuchos::SerialDenseMatrix<int, double>& A,
     std::vector<double> xv0, std::vector<double> yv0, double GridSize, double CompositeYoungs,
-    int systemsize)
+    double CompositePoissonsRatio, int systemsize, bool PressureGreenFunFlag)
 {
-  double r, pi = atan(1) * 4, raggio = GridSize / 2;
-  double C = 1 / (CompositeYoungs * pi * raggio);
-
-#pragma omp parallel for schedule(static, 16)  // Always same workload -> static
-  for (int i = 0; i < systemsize; i++)
+  double pi = atan(1) * 4;
```

Comment:
In the standard library header `#include <math.h>` most common mathematical constants are defined, i.e., $\pi=$`M_PI`. No need to compute it via the `atan` function.